### PR TITLE
Fix duplicates symbols when building with lld

### PR DESCRIPTION
**Pull request by MastaG**
Samba: https://svnweb.freebsd.org/ports/branches/2018Q4/net/samba47/files/patch-source3__wscript_build?view=markup&pathrev=481551
Wayland: https://gitlab.freedesktop.org/wayland/wayland/commit/c70fd8a812d74b5ef6fcba1287b409f4f8821e91

### CODE REVIEW

File: recipes-graphics/wayland/wayland_1.16.%.bbappend [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+SRC_URI_append += " file://fix_duplicate_main.patch "
```
Comment (kraj) [inline recipes-graphics/wayland/wayland_1.16.%.bbappend]: Usually we do not use `+=` and `_append` together, its better to use
`SRC_URI_append = " file://fix_duplicate_main.patch "`

File: recipes-connectivity/samba/samba_4.8.%.bbappend [hunk 2]
```diff
@@ -0,0 +1,3 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+SRC_URI_append += " file://fix_duplicate_pdb_search_init.patch "
```
Comment (kraj) [inline recipes-connectivity/samba/samba_4.8.%.bbappend]: we do not use += and _append together, its better to use _append alone

**Comment by MastaG** [comment]
Thank you kraj, I've removed the + signs :)

**Comment by MastaG** [comment]
Hi @kraj 
OE master has switched to wayland 1.17 where the upstream patch has landed.
I'm currently unable to do a PR for a couple of days.
But you can remove the bbappend and patch for wayland 1.16  now :)

**Comment by kraj** [comment]
Yes done with https://github.com/kraj/meta-clang/pull/114

**Pull request merged**